modular_skyrat/modules/conveyor_sorter/code/conveyor_sorter.dm

```diff
@@ -15,16 +15,17 @@
 
 /obj/item/conveyor_sorter/examine(mob/user)
 	. = ..()
-	. += span_notice("Use it to place down a conveyor sorter, up to three.")
+	. += span_notice("Use it to place down a conveyor sorter, up to a limit.")
 	. += span_notice("Use Alt-Click to reset the sorting list.")
 	. += span_notice("Attack things to attempt to add to the sorting list.")
 
 /obj/item/conveyor_sorter/attack_self(mob/user, modifiers)
-	if(length(spawned_sorters) >= 3)
-		to_chat(user, span_warning("You may only have three spawned conveyor sorters!"))
+	if(length(spawned_sorters) >= 4)
+		to_chat(user, span_warning("You may only have four spawned conveyor sorters!"))
```

Comment:
tbh, it should be a variable instead

Reply:
ok slight issue with that i cant get it to do that easily with also having the improved sorter decal that has the additional directions, but i still did set it to use a var and im testing it currently and will push it but i still have to have it in  `/obj/item/conveyor_sorter/improved/attack_self(mob/user, modifiers)` too otherwise the incorrect decal gets placed

Reply:
https://files.ratchtnet.com/i/b98cbd